pkg/network/ebpf/c/protocols/java-tls-erpc.h

```diff
@@ -0,0 +1,86 @@
+#ifndef _JAVA_TLS_ERPC_H
+#define _JAVA_TLS_ERPC_H
+
+#include "bpf_helpers.h"
+#include "tracer.h"
+#include "tags-types.h"
+
+#define USM_IOCTL_ID 0xda7ad09
+
+enum erpc_message_type {
+    REQUEST,
+    CLOSE_CONNECTION
+};
+
+
+int __attribute__((always_inline)) handle_request(conn_tuple_t* connection, void *data) {
+    const bool val = true;
+    u32 bytes_read = 0;
+
+    //read the actual length of the message (limited by HTTP_BUFFER_SIZE)
+    if (0 != bpf_probe_read_user(&bytes_read, sizeof(bytes_read), data)){
+        u64 pid_tgid = bpf_get_current_pid_tgid();
```

Comment:
add the pids lines under `#ifdef DEBUG`
so you can remove the forced if dependency